build/media-suite_compile.sh

```diff
@@ -1572,7 +1575,7 @@ if  { ! mpv_disabled vapoursynth || enabled vapoursynth; }; then
             gendef - "../$_file.dll" >/dev/null 2>&1 |
                 sed -r -e 's|^_||' -e 's|@[1-9]+$||' > "${_file}.def"
             dlltool -y "lib${_file}.a" -d "${_file}.def" \
-                $([[ $bits = 32bit ]] && echo "-U") 2>/dev/null
+                "$([[ $bits = 32bit ]] && echo "-U")" 2>/dev/null
```

Comment:
Isn't this going to be an empty parameter if `$bits = 64bit`?

Reply:
I think so too, might need a better way or just ignore the "prevent word splitting" warning

Reply:
Check these case-by-case. Most cases don't like empty parameters, like in this case. This needs another way of improving it, or just ignore the issue.

Reply:
Yea, going to just ignore it since dlltool doesn't seem to like it

Reply:
Still haven't removed the quotes here.

Reply:
I don't think the inside echo quotes would matter

Reply:
Nevermind, I didn't see it was fixed in a later commit.